libraries/AP_HAL_Linux/RCOutput_Bebop.cpp

```diff
@@ -283,6 +283,37 @@ void RCOutput_Bebop::_play_sound(uint8_t sound)
     _dev->get_semaphore()->give();
 }
 
+/*
+ * pwm is the pwm power used for the note.
+ *  It has to be >= 3, otherwise it refers to a predefined song
+ * (see _play_sound function)
+ * period is in us and duration in ms.
+ */
+void RCOutput_Bebop::play_note(uint8_t pwm, uint16_t period, uint16_t duration)
```

Comment:
since period and duration have different units, please name them like:  `period_usec`, `duration_msec` so it's clear their meaning for people reading the function signature and not the comment above.